topscroll/topscroll.js

```diff
@@ -110,8 +111,16 @@ try {
         });
       } catch (error) {
         // fallback without animation
-        window.top.scrollTo(0, imageDivHeight);
+        window.top.scrollTo(0, creativeHeight);
       }
+
+      // some browsers like ie11 thinks it supports scrollIntoView, because it does without options.
+      // so the try dosent fail but scrollIntoView didnt move the page at all, therefore this little test
+      setTimeout(function () {
+        if (document.documentElement.scrollTop === currentScrollPos) {
+          window.top.scrollTo(0, creativeHeight);
+        }
+      }, 1000);
```

Comment:
Det giver en ret irriterende effekt. Hvis man klikker på pilen, og begynder at scroll ned ad siden – så bliver man, efter 1 sekund, sendt tilbage til toppen.

Reply:
grunden til den nye fallback her er at IE tror den kan `scrollIntoView` og derfor ikke kommer ned i `catch`. Men kan godt se hvad du mener. nogen bud til et alternativ?

Reply:
ps. ser ud til at topscroll slet ikke virker i IE hos jer?

Reply:
Jeg tester slet ikke IE, vi har meget få brugere der bruger IE og jeg har ikke windows PC til at teste det :D 
Ser det helt skidt ud i IE? Vi har ikke fået klager, men da også vi testede det i IE da det gik live. 

Der er flere alternativer. 
Vi kunne evt. tjekke om browseren supportere options, eller måske bare altid kalde scrollIntoView uden options, hvilket heller ikke er nice, men bedre end det her :)

Reply:
der kommer slet ikke topscroll i IE hos jer
man kan desværre ikke teste for options, var det første jeg undersøgte, men må se om jeg kan finde på noget bedre

Reply:
hvad med at jeg også tester om der er scrollet over banneret som dette?
```
if (
          testScrollPos === currentScrollPos &&
          testScrollPos < creativeHeight
     ) {
          window.top.scrollTo(0, creativeHeight);
     }
```